deno/fhirchat-demo/islands/ChatIsland.tsx

```diff
@@ -25,32 +32,69 @@ export default function ChatIsland() {
     }
   }
 
+  const handleUserKeyPress = (event: KeyboardEvent) => {
+    const { key } = event;
+    if (key === "Enter" && question.value !== "") handleSubmit(event);
```

Comment:
I think this could be simplified to:

```suggestion
    const { key } = event;
    const allowSubmission = key === "Enter" && question.value !== "";
    if (allowSubmission) {
        handleSubmit(event);
    }
```
The return can be omitted because it's already at the end. The variable extraction is just preference for me though